File: reference/src/com/db4odoc/callbacks/TestObject.java
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */
package com.db4odoc.callbacks;

public class TestObject extends CountedObject{
	String name;	
	
	public TestObject(String name) {
		this.name = name;
	}
	
	public String toString() {
		return name+"/"+id;
	}
	
}

File: reference.vb.net/Db4objects.Db4odoc/Callbacks/TestObject.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Namespace Db4objects.Db4odoc.Callbacks

    Class TestObject
        Inherits CountedObject
        Private _name As String

        Public Sub New(ByVal name As String)
            _name = name
        End Sub

        Public Overloads Overrides Function ToString() As String
            Return _name + "/" + _id.ToString()
        End Function
    End Class
End Namespace